test/index.test.ts

```diff
@@ -6,11 +6,19 @@ Object.defineProperty(window.document, "referrer", {
   value: origin,
   writable: true,
 });
+
+Object.defineProperty(window, "location", {
+  value: {
+    href: origin + "?id=appid",
+  },
+  writable: true,
+});
+
 // eslint-disable-next-line
 import { actions, DispatchResponseEvent, createApp } from "../src";
 
 describe("createApp", () => {
-  const domain = "http://test-domain.com/?id=appid";
+  const domain = "http://test-domain.com/";
```

Comment:
I know that nobody recall any usage of `createApp` with provided `domain`, but before @Cloud11PL changes there was just a domain here, and now we're providing url with schema na host, which in that test make it incompatible with `createApp` execution without `domain` provided.